django/db/backends/postgresql/operations.py

```diff
@@ -257,3 +258,22 @@ def window_frame_range_start_end(self, start=None, end=None):
                 'and FOLLOWING.'
             )
         return start_, end_
+
+    def explain_query_prefix(self, output_format=None, **kwargs):
+        prefix = super().explain_query_prefix(output_format, **kwargs)
+
+        extra = {}
+        if output_format:
+            extra['FORMAT'] = output_format
+
+        if kwargs:
+            extra.update({
+                name.upper(): 'true' if value else 'false'
+                for name, value in kwargs.items()
+            })
+
+        if extra:
+            params = ', '.join('%s %s' % (key, value) for key, value in extra.items())
+            prefix += '(%s)' % params
```

Comment:
My concern with this is that there is no validation of options provided (as already mentioned for MySQL.)

~~Worse is that this is potentially vulnerable to SQL injection attacks as `value` could be user-provided input...~~
Just seen that options are converted to `'true'` or `'false'` above (which works as all options are currently booleans).